docs/troubleshooting.md

```diff
@@ -0,0 +1,19 @@
+# Troubleshooting
+## Permissions
+- `401 authz.invalid_permission` on specific endpoint
+1. Check list of your permissions on barong seeds.yml or tower admin panel
+2. If there are no such permission just add new permission on tower admin panel
+
+- `401 authz.invalid_permission` after login
+1. Check your permissions on barong rails console 
+```ruby
+irb(main)013:0> Permission.all
+irb(main)013:0> Rails.cache.read('permissions')
+```
+2. If it is wrong/empty just update it using following command
```

Comment:
its not an update.
I guess we need to explain mechanism a bit, as its a bit not clear